build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group 'com.github.hmcts'
-version '2.4.10'
+version '2.4.11'
```

Comment:
If I wanted to be picky, I don't think the amount of changes are a minor on the versioning

Reply:
Thanks Harpreet, updated version number as 2.5.0